build_tools/github_actions/build_configure.py

```diff
@@ -53,10 +50,24 @@ def build_configure():
         f"-DTHEROCK_PACKAGE_VERSION='{package_version}'",
         "-DCMAKE_C_COMPILER_LAUNCHER=ccache",
         "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache",
+        "-DTHEROCK_VERBOSE=ON",
+        "-DBUILD_TESTING=ON",
     ]
 
     # Adding platform specific options
-    cmd += platform_options[PLATFORM]
+    cmd += platform_options.get(PLATFORM, [])
+
+    if PLATFORM == "windows":
+        # VCToolsInstallDir is required for build. Throwing an error if environment variable doesn't exist
+        if not vctools_install_dir:
+            raise Exception(
+                "Environment variable VCToolsInstallDir is not set. Exiting."
+            )
```

Comment:
This could suggest configuring MSVC: https://github.com/ROCm/TheRock/blob/main/docs/development/windows_support.md#important-tool-settings

(fine to leave as is though)